protocol/invocation/rpcinvocation.go

```diff
@@ -40,8 +40,10 @@ type RPCInvocation struct {
 	reply           interface{}
 	callBack        interface{}
 	attachments     map[string]string
-	invoker         protocol.Invoker
-	lock            sync.RWMutex
+	// Refer to dubbo 2.7.6.  It is different from attachment. It is used in internal process.
+	attributes map[string]interface{}
```

Comment:
@cvictory pls remember that: when u define a map/slice/chan in a struct , u should init it by `make(map[string]interface{}, 8)` in NewRPCInvocation. If not, the program will panic when u visit or write this map.

Reply:
I have fix this issue in NewRPCInvocation function.